.github/workflows/github-actions-create-release.yml

```diff
@@ -0,0 +1,66 @@
+name: GitHub Actions - Create Release
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+
+jobs:
+  CreateRelease:
+    name: Create release for github-actions package
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.head.ref == 'release/actions' && github.event.review.state == 'approved' }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Create release
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require( 'fs' );
+            const workspace = '${{ github.workspace }}';
+            const packageDir = 'packages/js/github-actions';
+            const { default: PackageTool } = await import( `${ workspace }/${ packageDir }/utils/package-tool.js` );
+
+            const packageTool = new PackageTool( packageDir );
+            const { version } = packageTool.getSettings();
+            const { content } = packageTool.getChangelogByVersion( version );
+            const tag = `actions-v${ version }`;
+            const name = `github-actions ${ version }`;
+            const body = `## What's Changed\n${ content }`;
+            let release;
+
+            try {
+              const response = await github.rest.repos.createRelease( {
+                ...context.repo,
+                tag_name: tag,
+                target_commitish: context.payload.pull_request.head.ref,
+                name,
+                body,
+              } );
+              release = response.data;
+            } catch ( e ) {
+              await github.rest.pulls.dismissReview( {
+                ...context.repo,
+                pull_number: context.payload.pull_request.number,
+                review_id: context.payload.review.id,
+                message: 'Release creation workflow failed',
+              } );
+              throw e;
+            }
+
+            const commentBody = `New release has been created: [${ release.name }](${ release.html_url })`;
+            await github.rest.issues.createComment( {
+              ...context.repo,
+              issue_number: context.payload.pull_request.number,
+              body: commentBody,
+            } );
+
+            fs.writeFileSync( '/tmp/release.json' , JSON.stringify( release ), 'utf8' );
```

Comment:
:nail_care: Can we move those scripts to separate files? to get better syntax highlighting, lining, etc.

Reply:
Sounds good! I separated them in https://github.com/woocommerce/grow/pull/22/commits/c9a3deda008dbf153892ceabe7a80171c5f58b01.